colorama/ansitowin32.py

```diff
@@ -80,12 +88,41 @@ def __init__(self, wrapped, convert=None, strip=None, autoreset=False):
         # create the proxy wrapping our output stream
         self.stream = StreamWrapper(wrapped, self)
 
+        conversion_supported = True
         on_windows = os.name == 'nt'
         # We test if the WinAPI works, because even if we are on Windows
         # we may be using a terminal that doesn't support the WinAPI
         # (e.g. Cygwin Terminal). In this case it's up to the terminal
         # to support the ANSI codes.
-        conversion_supported = on_windows and winapi_test()
+        if on_windows and platform.release().startswith('10'):
+            # Windows 10 supports ANSI sequences using virtual terminal mode.
+            # However, this has been turned off by default from an early Windows 10 version.
+            # We turn virtual terminal support back on, then set
+            # `conversion_supported` to False. Behaviour will then be the same
+            # as for Linux - colorama doesn't use win32 calls.
+            from ctypes import wintypes, byref  # import wintypes now we know we're on Windows.
```

Comment:
Please move relevant code to win32.py, that's where we check the imports and define the ctypes functions.

Reply:
After reading #300 I checked through the outstanding PRs. #139 by @segevfiner has used win32.py. That PR also enables vt via a function in winterm.py (as you suggested I should do). Once I've set up appropriate test environments (old Windows 10, Cygwin, etc), I'll run the same tests using #139. That may be a better candidate for implementing this functionality. @segevfiner identified vt as a more appropriate solution while I was still trying to make win32 calls behave.